kairos-cli/bin/crypto/error.rs

```diff
@@ -0,0 +1,10 @@
+use thiserror::Error;
+
+#[derive(Error, Debug)]
+pub enum CryptoError {
+    /// Failed to parse a public key from a formatted string.
```

Comment:
This comment is not necessary as it repeats the code

Reply:
This is intended for documentation:

![image](https://github.com/cspr-rad/kairos/assets/6506780/4feaeacc-ce27-4354-8c50-90a382608242)